### PR TITLE
Transfer the logic from php to here

### PR DESCRIPTION
**Pull request by Arpita-Jaiswal**

### CODE REVIEW

**Comment by amitu** [comment]
@Arpita-Jaiswal lets not checking the wasm file, but create an action that builds and uploads it.

File: backend/src/login.rs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    host: ft_sdk::Host,
+) -> ft_sdk::processor::Result {
+    let next = next.unwrap_or_else(|| {
+        let order_query = order
+            .filter(|order| !order.is_empty())
+            .map(|order| format!("?order={}", order))
+            .unwrap_or_default();
+
+        format!("/{}", order_query)
+    });
```
Comment (amitu) [inline backend/src/login.rs]: Write a comment for such code, show input/ouput. Better move it to a function, and write some meaningful function name, and some basic rust test to one can see inputs and outputs.

File: Cargo.toml [hunk 2]
```diff
@@ -22,18 +22,21 @@ homepage = "https://www.fifthtry.com"
 
 
 [workspace.dependencies]
-ft-sdk = { version = "0.1.11", features = ["field-extractors"] }
+ft-sdk = { features = ["field-extractors", "auth-provider"], git = "https://github.com/FifthTry/ft-sdk.git", branch = "add_expires_at_session" }
```
Comment (amitu) [inline Cargo.toml:25]: I think using path is more developer friendly here, as you will not have to a do a commit in ft-sdk before you can test if it compiles. Why are you not using path?

File: backend/src/register.rs [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+/// Creates a new session with a specified expiration duration.
+fn create_new_session(
+    conn: &mut ft_sdk::Connection,
+    user_id: &ft_sdk::auth::UserId,
+    session_id: Option<ft_sdk::auth::SessionID>,
+) -> Result<ft_sdk::auth::SessionID, ft_sdk::Error> {
+    // Define the duration for which the session should be valid.
+    let session_expiration_duration = Some(chrono::Duration::days(todayhasbeen::DURATION_TO_EXPIRE_ACCESS_TOKEN_IN_DAYS));
```
Comment (amitu) [inline backend/src/register.rs:156]: Remove all access token expiry logic. Every site has expiry, expiry for a site is a fixed value (it usually does not differ view to view), so it must be controlled by an environment variable. We will add this feature later on, so for now whatever is the ft-sdk cookie expiry is can be used.

**Comment by Arpita-Jaiswal** [comment]
Not needed anymore

**Pull request closed without merging**